src/components/loaders/UpdateLoader.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+
+const UpdateLoader = () => (
+  <div className='update'>
+    <div className='loader'><span /><span /><span /></div>
```

Comment:
Why do we have all these spans?

Reply:
They are used to generate the loader. Each of them is styled as a round blue dot